test-engineering/contract-tests/client/tests/conftest.py

```diff
@@ -22,19 +23,28 @@ def pytest_configure(config):
         Scenario(**scenario) for scenario in loaded_scenarios["scenarios"]
     ]
 
-    # Check that all 200 OK responses in test scenarios contain correct tiles
+    # Check that all 200 OK responses in test scenarios contain correct
```

Comment:
Change applied: [6e5ad37](https://github.com/mozilla-services/contile/pull/422/commits/6e5ad37f279a1bd02cb69911aa0c2754d146df3a)

I like this, it's much cleaner! 
I added typing and used the `get` method instead of an index, since an IDE issues an error.
<img width="929" alt="image" src="https://user-images.githubusercontent.com/3422688/180341521-a5e55a13-73ce-4ec4-8e24-9bd30d6641c6.png">

Note: The IDE issues a warning for`SERVICE_MODEL`. I think this is a matter of preference, so left it as is.
<img width="480" alt="image" src="https://user-images.githubusercontent.com/3422688/180342278-4b29a3e9-f3f4-430e-b628-0024aa1e2f7a.png">

Reply:
>     1. Why replace pytest.UsageError with RuntimeError?

Oops, I forgot to update that in my review comment. I tried this out locally outside of pytest, so I replaced that with a `RuntimeError`. Please raise a `pytest.UsageError`. 😁 

>     2. I added typing and used the `get` method instead of an index, since the IDE issues an error.

Hmm that doesn't seem correct to me. Using `get` means we don't raise an `KeyError` for unknown services. Does PyCharm still raise that error if you specify the types of values in the dict?

```python
    SERVICE_MODEL: Dict[Service, Union[Records, Tiles]] = {
        Service.PARTNER: Records,
        Service.CONTILE: Tiles,
    }
```

>     3. Note: The IDE issues a warning for `SERVICE_MODEL`. I think this is a matter of preference, so left it as is.

You can move `SERVICE_MODEL` to the module scope. I assume that will make the warning in PyCharm go away. I used all-uppercase because it's a constant. Perfectly fine to define that outside of the function.

Reply:
Change applied: Change applied: [529afcf](https://github.com/mozilla-services/contile/pull/422/commits/529afcf1b66575b0890b80c9498ab0aedcb6a112)

1. RuntimeError replaced with pytest.UsageError 
2. SERVICE_MODEL type defined as Dict[Service, Union[Records, Tiles]] with `get` instead of index to prevent the PyCharm error message. 
3. SERVICE_MODEL moved to module scope